### ISSUE TITLE
[BUG] Circular Import Error when Running Example

### ISSUE THREAD

**Issue by ilteralp** [neuralhydrology/neuralhydrology#157]
Hi all, thank you very much for creating this repo. I'm currently facing an issue while attempting to run the `adding-gru.ipynb` example located at `/examples/02-Adding-Models/adding-gru.ipynb`. When I execute the import cell, I encounter the following error:

```
ImportError: cannot import name 'get_model' from partially initialized module 'neuralhydrology.modelzoo' (most likely due to a circular import) (d:\melike\repos\neuralhydrology\neuralhydrology\modelzoo\__init__.py)
```

Here is the detailed stack trace of the error:

```
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
Cell In[1], line 7
      4 import torch
      5 from torch import nn
----> 7 from neuralhydrology.modelzoo import get_model
      8 from neuralhydrology.modelzoo.head import get_head
      9 from neuralhydrology.modelzoo.basemodel import BaseModel

File d:\melike\repos\neuralhydrology\neuralhydrology\modelzoo\__init__.py:5
      1 import warnings
      3 import torch.nn as nn
----> 5 from neuralhydrology.modelzoo.arlstm import ARLSTM
      6 from neuralhydrology.modelzoo.cudalstm import CudaLSTM
      7 from neuralhydrology.modelzoo.customlstm import CustomLSTM

File d:\melike\repos\neuralhydrology\neuralhydrology\modelzoo\arlstm.py:10
      7 import torch
      8 import torch.nn as nn
---> 10 from neuralhydrology.modelzoo.basemodel import BaseModel
     11 from neuralhydrology.modelzoo.inputlayer import InputLayer
     12 from neuralhydrology.modelzoo.head import get_head

File d:\melike\repos\neuralhydrology\neuralhydrology\modelzoo\basemodel.py:6
      3 import torch
      4 import torch.nn as nn
----> 6 from neuralhydrology.evaluation.utils import load_scaler
      7 from neuralhydrology.utils.config import Config
      8 from neuralhydrology.utils.samplingutils import sample_pointpredictions, umal_extend_batch

File d:\melike\repos\neuralhydrology\neuralhydrology\evaluation\__init__.py:3
      1 from pathlib import Path
----> 3 from neuralhydrology.evaluation.tester import BaseTester, RegressionTester, UncertaintyTester
      4 from neuralhydrology.utils.config import Config
      7 def get_tester(cfg: Config, run_dir: Path, period: str, init_model: bool) -> BaseTester:

File d:\melike\repos\neuralhydrology\neuralhydrology\evaluation\tester.py:23
     21 from neuralhydrology.evaluation.metrics import calculate_metrics, get_available_metrics
     22 from neuralhydrology.evaluation.utils import load_scaler, load_basin_id_encoding, metrics_to_dataframe
---> 23 from neuralhydrology.modelzoo import get_model
     24 from neuralhydrology.modelzoo.basemodel import BaseModel
     25 from neuralhydrology.training import get_loss_obj, get_regularization_obj

ImportError: cannot import name 'get_model' from partially initialized module 'neuralhydrology.modelzoo' (most likely due to a circular import) (d:\melike\repos\neuralhydrology\neuralhydrology\modelzoo\__init__.py)
```
Thank you very much!

**Comment by kratzert** [comment]
Thanks, we are aware of this issue and looking into this. Hopefully I can update you soon with more info.

**Comment by kratzert** [comment]
Should be fixed in #161. Would be great if you could confirm this @ilteralp

**Comment by ilteralp** [comment]
Yes, it's working now. Thanks a lot!
